session.go

```diff
@@ -43,3 +44,12 @@ func (s *Session) FlushStorage() (err error) {
 	_, err = synchronousEvent(s.ctx, s, s.w, Event{Name: EventNameSessionCmdFlushStorage, TargetID: s.id}, EventNameSessionEventFlushedStorage)
 	return
 }
+
+// Loads a chrome extension
+func (s *Session) LoadExtension() (err error) {
+	if err = s.ctx.Err(); err != nil {
+		return
+	}
+	_, err = synchronousEvent(s.ctx, s, s.w, Event{Name: EventNameSessionCmdLoadExtension, TargetID: s.id})
```

Comment:
[Here](https://github.com/asticode/go-astilectron/blob/master/event.go#L41) you will also need to add a field

```go
Path string `json:"path"`
```

(Make sure fields are ordered alphabetically)

And in the `synchronousEvent` call, the body will have to look like

```go
Event{Name: EventNameSessionCmdLoadExtension, Path: path, TargetID: s.id}
```

so that path is sent to JS.